README.md

```diff
@@ -3,6 +3,10 @@
 [![Downloads](https://img.shields.io/github/downloads/99designs/aws-vault/total.svg)](https://github.com/99designs/aws-vault/releases)
 [![Continuous Integration](https://github.com/99designs/aws-vault/workflows/Continuous%20Integration/badge.svg)](https://github.com/99designs/aws-vault/actions)
 
+> [!WARNING]
+> This project has been abandoned and it's not receiving any more updates. If you want to continue to receive updates
+> or contribute, please feel free to look at the active fork at: https://github.com/ByteNess/aws-vault
```

Comment:
there are currently multiple forks.
if you can add a list of alternatives, I'll approve

Reply:
@FernandoMiguel shall we merge this now or do you want to add something else?

Reply:
Anything else we need for this PR?